pkg/authorization/authorizer/scope/converter.go

```diff
@@ -210,10 +211,18 @@ func (userEvaluator) ResolveGettableNamespaces(scope string, clusterPolicyGetter
 // escalatingScopeResources are resources that are considered escalating for scope evaluation
 var escalatingScopeResources = []unversioned.GroupResource{
```

Comment:
These changes do not look correct.

The lines that have multiple resources are missing the new groups.